src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -124,11 +145,11 @@ export class CourseManagementRequests {
     }
 
     /**
-     * add text exercise to exercise group in exam
+     * add text exercise to an exercise group in exam or to a course
      * @returns <Chainable> request response
      * */
-    addTextExerciseToExam(group: any, title: string) {
-        textExercise.exerciseGroup = group;
+    createTextExercise(title: string, course?: any, group?: any) {
+        const textExercise = this.getCourseOrExamExercise(textExerciseTemplate, course, group);
```

Comment:
Same suggestion as above
```suggestion
    createTextExercise(title: string, body: any, isExamExercise: boolean) {
        const textExercise = this.getCourseOrExamExercise(textExerciseTemplate, isExamExercise);
```